sklearn/svm/classes.py

```diff
@@ -1051,7 +1051,8 @@ def fit(self, X, y=None, sample_weight=None, **params):
         return self
 
     def decision_function(self, X):
-        """Distance of the samples X to the separating hyperplane.
+        """Signed distance to the separating hyperplane: positive if inlier,
+        negative if outlier.
```

Comment:
following https://www.python.org/dev/peps/pep-0257/ the first line should be short without line break.

I would write:


```
 """Signed distance to the separating hyperplane

Signed distance is positive for an inlier and negative for out outlier.

...
```